src/components/Home.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Linters helps you to deliver a clean & readable code , kindly remove all the disabling commands and try to fix them 🙏